bip-0088.mediawiki

```diff
@@ -227,3 +227,7 @@ Its representation after parsing can be:
 
 Its representation after parsing can be:
     [[(2147483648, 4294967295)], [(0, 0)]]
+
+==Acknowledgements==
+
+Special thanks to Peter D. Gray, Dr. Maxim Orlovsky, Robert Spigler and others for their feedback on the specification, Janine (github:@Enegnei) for the help in preparing the draft.
```

Comment:
nit: grammar.

Special thanks to Peter D. Gray, Dr. Maxim Orlovsky, Robert Spigler and others for their feedback on the specification, [and to] Janine (github:@Enegnei) for their help in preparing the draft.